cmd/kpod/common.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	is "github.com/containers/image/storage"
+	"github.com/containers/storage"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+var needToShutdownStore = false
+
+func getStore(c *cli.Context) (storage.Store, error) {
+	options := storage.DefaultStoreOptions
+	if c.GlobalIsSet("root") || c.GlobalIsSet("runroot") {
```

Comment:
@rhatdan @nalind could you shed some light on this OR case here? can we work with one root but not the other?